docs/input/docs/ci-systems/appveyor.md

```diff
@@ -0,0 +1,25 @@
+---
+Order: 10
+Title: AppVeyor
+Description: Running on AppVeyor
+---
+
+## specifics
+
+None.
+
+## example config
+
+```yaml
+image:
+  - Visual Studio 2019
+
+test: off
+build: off
+
+build_script:
+  - ps: .\build.ps1 --target=CI
+
+cache:
+  - "tools -> recipe.cake,tools/packages.config"
```

Comment:
@nils-a @AdmiringWorm Am I right in saying that going forward, we are going to be switching to smaller bootstrapper files that simply execute the Cake .Net Global Tool?  If so, I think we should include the same tools manifest here, and replace the packages.config file.  Thoughts?  From a consistency point of view, I think we should recommend usage of the .Net Global Tool everywhere.

Reply:
I am not against it, or to have a combination of both (like what is in #625, or I can update that one to use a smaller bootstrapper with only the global tool).

> If so, I think we should include the same tools manifest here, and replace the packages.config file. Thoughts?

IMO, as long as the https://github.com/cake-build/resources repository uses bootstrappers for .NET Framework/Mono, I think we perhaps should follow that in the examples.
We could however have a comment with an additional cache mentioning to use that instead when targeting the .NET Core global tool.

Reply:
@AdmiringWorm on the oher hand we're not providing samples for using cake, but cake.recipe. So I *think* it boils down to what is the cake.recipe-way of calling cake. I feel we could go forward here and promote the switch to using the global tool.

Reply:
@AdmiringWorm while the default bootstrappers in the resources repository haven't quite caught up yet, the default guidance going forward is going to be to use the .Net Global Tool.  This can be seen in the [setting up a new project](https://cakebuild.net/docs/getting-started/setting-up-a-new-project) article on the site, which @pascalberger updated to reflect the new guidance, so I think I am ok with having a much smaller bootstrapper being suggested to run the .Net Global Tool.  This would also mean that all CI's would be using the same version of Cake.

Reply:
I have updated the example to reflect the NET Core - version of the cache key.